### DISCUSSION TITLE
Climax keypad

### DISCUSSION THREAD

**Discussion by Sopis67** [Koenkk/zigbee2mqtt#17444] [Devices]
Hello,

I just bought a Climax keypad. It identifies itself as unsupported KP-ACE_00.00.03.11TC . 
Are there an easy way to change this to KP-ACE_00.00.03.12TC which is in the list of working devices.
I'm pretty sure it's the same, just with a branded name.

Thanks
Sopis67

**Comment by Koenkk** [comment]
Added!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) (and the next release on 1 May)
